src/AstroBase.jl

```diff
@@ -115,4 +115,64 @@ function tio_locator(jd1, jd2)
     -47e-6 * t * sec2rad(1)
 end
 
+"""
+    precession_rate_part_of_nutation(jd1, jd2)
+
+Returns precession corrections for a given 2 part Julian date (TT).
+
+# Example
+
+```jldoctest
+
+```
+"""
+function precession_rate_part_of_nutation(jd1, jd2)
+    PRECESSION = -sec2rad(0.29965)
```

Comment:
Move these constants out of the function?

Reply:
okay :)